### PR TITLE
[Tests-Only]POM implementation in the syncing tests

### PR DESCRIPTION
**Pull request by Talank**
### Description
This PR  implementing the Page Object Model in the client test for syncing.

### Related Issue
- https://github.com/owncloud/client/issues/8577

### CODE REVIEW

File: test/gui/shared/scripts/pageObjects/Activity.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import names
+import squish
+from objectmaphelper import RegularExpression
+from helpers.FilesHelper import buildConflictedRegex
+
+
+class Activity:
+    SUBTAB = {
```
Comment (swoichha) [inline test/gui/shared/scripts/pageObjects/Activity.py:8]: Can you rename it to something else. It is a bit hard to understand which tab/button is this.

Comment (Talank) [reply test/gui/shared/scripts/pageObjects/Activity.py:8]: The name of the tab depends upon the tab name passed here `squish.clickTab(squish.waitForObject(self.SUBTAB), tabName)` So, it is like a generic selector, not a specific one.

Comment (Talank) [reply test/gui/shared/scripts/pageObjects/Activity.py:8]: The name of the tab depends upon the tab name passed here squish.clickTab(squish.waitForObject(self.SUBTAB), tabName) So, it is like a generic selector, not a specific one.

**Review by swoichha — COMMENTED** [review]
Looks good :+1:

**Review by individual-it — APPROVED** [review]

**Pull request merged**